src/common/registry/test/key-registry/connect.e2e-spec.ts

```diff
@@ -21,11 +20,11 @@ describe('Registry', () => {
     process.exit(1);
   }
   const address = REGISTRY_CONTRACT_ADDRESSES[process.env.CHAIN_ID];
-  const operatorsWithModuleAddress = operators.map((key) => {
-    return { ...key, moduleAddress: address };
-  });
+  // const operatorsWithModuleAddress = operators.map((key) => {
+  //   return { ...key, moduleAddress: address };
+  // });
```

Comment:
probably, there lines not needed

Reply:
I updated the blockHash in the tests because the ELnode on Goerli didn't work with this hash. I left this test code commented out so it can be uncommented during the move to Holesky.